Meta/CMake/libweb_generators.cmake

```diff
@@ -106,23 +106,34 @@ function (generate_js_wrappers target)
 
     function(libweb_js_wrapper class)
         cmake_parse_arguments(PARSE_ARGV 1 LIBWEB_WRAPPER "ITERABLE" "" "")
+        cmake_parse_arguments(PARSE_ARGV 1 LIBWEB_WRAPPER "NO_INSTANCE" "" "")
```

Comment:
Not sure if this first commit will stay at the end of the PR, but NO_INSTANCE should be added to the first call.

`cmake_parse_arguments(PARSE_ARGV 1 LIBWEB_WRAPPER "ITERABLE;NO_INSTANCE" "" "")`

Reply:
Indeed, the idea is for this to be gone by the end of the PR :)